theano/gpuarray/elemwise.py

```diff
@@ -202,7 +184,7 @@ def c_init_code_struct(self, node, name, sub):
                            typecode=o.type.typecode)
```

Comment:
line 177 should probably be changed too. now outs have the same number of element then node.outputs. The variable p is probably not needed anymore.